demisto_sdk/commands/common/update_id_set.py

```diff
@@ -869,8 +869,8 @@ def process_integration(file_path: str, print_logs: bool) -> list:
                 res.append(get_integration_data(file_path))
         else:
             # package integration
-            package_name = os.path.basename(file_path)
-            file_path = os.path.join(file_path, '{}.yml'.format(package_name))
+            file_path = glob.glob(os.path.join(file_path, '*yml'))
```

Comment:
better to fix the problematic content and not change this code.
we already have this assumption in many places.

using glob can cause problems when used locally:
* what if you have a unified file?
* what if you have leftovers?

Reply:
but the original code was incorrect as well as we do support `integration-name.yml` as a valid format that was not caught.
If we are to fix content in such a way it might be better to solve this issue: https://github.com/demisto/etc/issues/29240
in the meantime, I think we should open an issue to solve this and currently fix this in such a way that let's us pass the build correctly (as you said the main issue with this is more than likely to be the local runs rather than the build runs).

One minor improvement we can implement right now is to go through the list and take out anything that ends with `_unified.yml` solving at least part of the problem.

@yaakovi @ChanochShayner - What do you think?